src/main/rx/crsf.c

```diff
@@ -293,6 +293,13 @@ static void handleCrsfLinkStatisticsTxFrame(const crsfLinkStatisticsTx_t* statsP
         setLinkQualityDirect(stats.uplink_Link_quality);
     }
 #endif
+
+    if (debugMode == DEBUG_CRSF_LINK_STATISTICS_UPLINK) {
```

Comment:
I know this is done incorrectly in other places in this file, but it is not necessary to explicitly check the `debugMode` before the use of `DEBUG_SET()` - this is implicitly done in `DEBUG_SET()`: https://github.com/betaflight/betaflight/blob/a53c6dd8ccd71ec3fe0eb819d2c9ab64bdf1a024/src/main/build/debug.h#L27

Reply:
Oh, didn't realise that. I will fix all these.
Thanks!